core/modules/layout/layout.module

```diff
@@ -2379,7 +2411,7 @@ function layout_layout_info() {
       'flexible' => TRUE,
       'type' => 'layout',
       'path' => 'templates',
-      'preview' => 'flexible_template.png',
+      'preview' => 'pencil-ruler',
```

Comment:
I do not love this :)

Reply:
This is only temporary. The plan is to have preview .svg icons for flexible layout templates be automatically generated as part of https://github.com/backdrop/backdrop-issues/issues/4104 ...besides, the current preview icon we have does not provide any realistic representation of the regions either.